.github/scripts/generate-packages-linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+# Build scala-cli native image
+./mill -i show cli-core.nativeImage
+# Build deb package for scala-cli
+cs launch org.virtuslab::scala-packager-cli:0.1.3 -- --source-app-path ./out/cli-core/nativeImage/dest/scala --output scala-cli.deb --debian
+# Copy deb package to artifacts
+mkdir artifacts && cp ./scala-cli.deb artifacts/scala-cli.deb
```

Comment:
Thanks to `set -e`, we don't need the `&&` anymore. And passing `-p` to `mkdir` is safer, in case the directory already exists (if we move this step around for example).
```suggestion
mkdir -p artifacts
cp ./scala-cli.deb artifacts/scala-cli.deb
```